### DISCUSSION TITLE
Is it possible to render the pointerStrip and call getPointerProps on first press?

### DISCUSSION THREAD

**Discussion by tverm94** [Abhinandan-Kushwaha/react-native-gifted-charts#934] [General]
I have a line chart that leverages getPointerConfig and getPointerProps to allow a user to drag their finger along a graph and see the values. The only issue is that, getPointerProps doesnt get called until you start dragging, whereas I want it to begin on first press.

So for example - you press at the index 50, nothing happens, once you drag to 49 or 51, getPointerProps fires and the pointer line shows.

Is there a way I can get this functionality on first press?

**Comment by Abhinandan-Kushwaha** [comment]
Yes @ThomasLVermeersch It is possible now in the latest release. Please update the library.

I have added a property named `activatePointersInstantlyOnTouch` inside the `pointerConfig` prop and its default value is true.
